### PR TITLE
Aluminum recycling.

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**

### CODE REVIEW

**Comment by mugling** [comment]
Been suggested before. Smelting aluminiun is extremely energy intensive and a complex industrial process.

Instead of this we need to create some (new) locations where ingots spawn

**Comment by BorkBorkGoesTheCode** [comment]
This recipe uses aluminum foil or soda cans

**Comment by BorkBorkGoesTheCode** [comment]
Search for how to melt soda cans

**Comment by BorkBorkGoesTheCode** [comment]
@mugling  This is not the Bayer Process
https://en.wikipedia.org/wiki/Bayer_process

This is melting down recycled aluminum, which is far easier.
https://en.wikipedia.org/wiki/Aluminium_recycling

**Comment by BorkBorkGoesTheCode** [comment]
Please review my changes before dismissing them.

**Comment by Mecares** [comment]
The numbers do not seem to add up, an aluminum ingot weights 2 kg while a can weights only 10g according to the item browser, it would require 200 cans or 500 foil to avoid aluminum appearing from thin air. I think it would be more elegant to make aircraft wrecks spawn something like mangled aluminum that could be melted instead.

**Comment by BorkBorkGoesTheCode** [comment]
I just checked the source code, soda cans weigh 13 grams. So it will take 154 cans.

**Comment by Mecares** [comment]
Yeah the item browser is rounding

**Comment by Cyrano7** [comment]
If this was to be implemented another idea is to add a scrap aluminum. It could be added to the deconstruction of some items. The heavy duty flashlight is a good one. It currently gives you aluminum ingots when disassembled which is a unrealistic. Also the size of the ingot could be reduced.

**Comment by BorkBorkGoesTheCode** [comment]
I would love to split scrap metal into real-world elements.

**Comment by BorkBorkGoesTheCode** [comment]
I have a prototype of scrap aluminum.

**Comment by codemime** [comment]
Aluminum cables and wires are rich sources of scrap aluminum. It would be nice to [have them](https://en.wikipedia.org/wiki/Overhead_power_line) in the game.

**Comment by ifreund** [comment]
Also, some bicycles would give a lot of scrap aluminum.

**Comment by kevingranade** [comment]
Please look into the properties of melted aluminum scrap, or the history of
this issue (search "aluminum ingot" for instance).

tl;dr If you simply melt down aluminum scrap, you get nearly useless soft
aluminum, not the structural aluminum you're expecting and that is
represented by the aluminum ingot item. If you want to add "non-structural
aluminum", and associated crafting recipes, feel free. If you want to
obtain structural aluminum, put it somewhere harvestable.
